src/main/java/org/jabref/logic/migrations/MergeReviewIntoCommentMigration.java

```diff
@@ -48,7 +48,7 @@ public void performConflictingMigration(ParserResult parserResult) {
     private String mergeCommentFieldIfPresent(BibEntry entry, String review) {
         if (entry.getField(FieldName.COMMENT).isPresent()) {
             LOGGER.info(String.format("Both Comment and Review fields are present in %s! Merging them into the comment field.", entry.getAuthorTitleYear(150)));
-            return String.format("%s\n%s:\n%s", entry.getField(FieldName.COMMENT).get().trim(), Localization.lang("Review"), review.trim());
+            return String.format("%s\n%s:\n%s", entry.getField(FieldName.COMMENT).get(), Localization.lang("Review"), review);
```

Comment:
Maybe, `%s` does something magic? Or does the `get()` do magic?

Reply:
wow. well I'll write a unit test for that and then we'll see.

Reply:
I always forget the specifc string arguments formatter, too
`%$1s` etc- Maybe that helps?
An argument index is specified as a number ending with a “$” after the “%” and selects the specified argument in the argument list.
https://dzone.com/articles/java-string-format-examples

Reply:
No, we just forgot that `COMMENT` is a mutli line field. We decided two or three years ago that JabRef rewraps all field values to have a good looking bibtex file. We make exceptions for **predefined** multi line fields. I am working on an update.